nat-lab/bin/dns-server

```diff
@@ -6,10 +6,12 @@ from twisted.names import dns, server
 KNOWN = {
     dns.AAAA: {
         b"google.com": dns.Record_AAAA(address="2a00:1450:400e:80c::200e"),
+        b"www.google.com": dns.Record_AAAA(address="2a00:1450:400e:80c::200e"),
```

Comment:
Just an idea - maybe instead of two entries we could use `*.google.com` ?

Reply:
This is very basic server and the logic is just a lookup in a hashtable - I don't think there is a simple way to handle suffix match like that.